config/config.go

```diff
@@ -40,12 +47,14 @@ type ArtifactoryConfig struct {
 
 // DownloaderConfig holds the configuration for the various downloaders
 type DownloaderConfig struct {
-	Type      string `mapstructure:"type"`
-	AwsBucket string `mapstructure:"aws_bucket"`
-	AwsPrefix string `mapstructure:"aws_prefix"`
-	AwsKey    string `mapstructure:"aws_key"`
-	AwsSecret string `mapstructure:"aws_secret"`
-	AwsRegion string `mapstructure:"aws_region"`
+	Type        string `mapstructure:"type"`
```

Comment:
I think we might want to break this struct into multiple structs by downloader type.